app/src/main/res/layout/gradeitem_node.xml

```diff
@@ -8,23 +8,32 @@
     android:orientation="horizontal">
 
     <TextView
+        android:background="@color/leaf_background"
+        android:foreground="?android:attr/selectableItemBackground"
+        android:textColor="@color/sakai_light_black"
         android:id="@+id/txt_assignment_name"
         android:layout_width="260dp"
         android:layout_height="wrap_content"
         android:layout_marginEnd="8dp"
         android:layout_marginRight="8dp"
-        android:paddingBottom="10dp"
-        android:paddingLeft="40dp" />
+        android:paddingBottom="8dp"
+        android:paddingLeft="40dp"
+        android:textSize="16sp"
+        android:paddingTop="8dp"/>
     
     <View
         android:layout_width="0dp"
         android:layout_height="0dp"
         android:layout_weight="1"/>
 
     <TextView
+        android:textSize= "16sp"
+        android:background="@color/leaf_background"
+        android:foreground="?android:attr/selectableItemBackground"
+        android:textColor="@color/sakai_light_black"
         android:id="@+id/txt_grade"
         android:layout_width="wrap_content"
-        android:layout_height="20dp"
+        android:layout_height="fill_parent"
```

Comment:
@akench fill_parent is deprecated, try just using wrap_content or something instead, or something that isn't deprecated

Reply:
I'll change it to match parent